lms/models/upload.py

```diff
@@ -23,22 +23,29 @@ def _is_uploaded_before(
 
 
 def _upload_to_db(
-        exercise_id: int,
+        exercise_number: int,
+        course_id: int,
```

Comment:
I know I did it, but it's a strange signature to a function.
We should either pass database objects or Python prototype objects.
I think we should go with prototypes. Do you mind changing `User` to an int?
(If you prefer to save it for another ticket that's OK :))

Reply:
Isn't it strange that I already have the `User` in the `new` function, and then I'll pass the `user_id` instead of the `User` to the `_upload_to_db` function, and then I need to get the `User` in this function again?

Reply:
Yeah it is a bit strange, but it's also strange to have function which receives both primitives and complex objects in the same function signature.

The component that calls the bridge (which are the `models/*.py` files) shouldn't be aware of the database, IMHO